ui/test/services/project_service_test.dart

```diff
@@ -192,6 +197,19 @@ void main() {
         expect(result.keys, ['high', 'low']);
       });
 
+      test('loads obligations', () async {
+        final dependencies = <Dependency>{};
+        dependencies.add(Dependency(id: dependencyId));
```

Comment:
This line can be combined with the previous line into:
```dart
final dependencies = {Dependency(id: dependencyId)};
```
And then you can probably also inline it into the next line, because the dependencies are not used anywhere else in the test.